app/experimenter/legacy-ui/templates/experiments/nimbus-ui.html

```diff
@@ -0,0 +1,32 @@
+{% load static %}
+{% load experiment_extras %}
+
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <link rel="icon" href="{% static "core/imgs/favicon.ico" %}" />
+  <meta name="description" content="" />
+  <meta name="referrer" content="origin" />
+  <meta name="robots" content="noindex,nofollow" />
+  <meta name="viewport"
+    content="width=device-width,initial-scale=1,minimum-scale=1,maximum-scale=2,user-scalable=yes" />
+  <title>Mozilla Experimenter</title>
+</head>
+
+<body>
+  <noscript>Mozilla Experimenter requires JavaScript to run.</noscript>
+  <div id="root" data-config="{{ APP_CONFIG|encode_uri }}"></div>
+
+  {% if debug %}
+    <script src="http://localhost:3000/static/js/bundle.js"></script>
+    <script src="http://localhost:3000/static/js/0.chunk.js"></script>
+    <script src="http://localhost:3000/static/js/main.chunk.js"></script>
+  {% else %}
+    {# TODO: this will not work as CRA generates multiple JS files #}
```

Comment:
Ack, yeah. When the build command is run react-scripts compiles everything into what looks like 3, but might vary, fingerprinted scripts that we'll need to render. Generally the CRA build process takes care of modifying the HTML file, but things are a little different for us. I suppose we'll need to traverse the build folder for `*.js` files and print them in to this template.

<img width="269" alt="Screen Shot 2020-10-06 at 8 25 46 PM" src="https://user-images.githubusercontent.com/6392049/95274003-a116ed80-0812-11eb-9534-7ab5569cd082.png">

Reply:
Naively noodling around this morning, could we glob it and render each returned file (would need to also modify each result to replace local path with URL I suppose):

```python
current = pathlib.Path().absolute()
listing = glob.glob(os.path.join(current, '../nimbus-ui/build/static/js/*.js'))
```

Reply:
Alternatively, it looks like [there](https://github.com/facebook/create-react-app/issues/3365#issuecomment-376546407) are [ways](https://stackoverflow.com/questions/49123097/generate-single-physical-javascript-file-using-create-react-app) to override or augment the create-react-app Webpack config if we really want to generate a single output file. We've done this in FxA, and it's a little tricky in some respects, but it sounds do-able.

Reply:
Yeah I'd think finding some way to force it to a single output file would be preferred long term solution instead of having to try to discover the randomly generated ones every time.

Reply:
Will try it out and report back :)

Reply:
Think I've got something working.

Reply:
Oh sick 🎉